packages/next/src/lib/turbopack-warning.ts

```diff
@@ -62,6 +74,12 @@ const prodSpecificTurboNextConfigOptions = [
   'experimental.outputFileTracingIgnores',
   'experiemental.outputFileTracingIncludes',
   'experimental.gzipSize',
+  'experimental.useDeploymentId',
+  'experimental.useDeploymentIdServerActions',
+  'experimental.deploymentId',
```

Comment:
Are we properly propagating all env variables expected to be set for these configs via `DefinePlugin` in turbopack now?

x-ref: https://github.com/vercel/next.js/blob/6123a9700b66d54a7cabe1a9ec23ada11f576a36/packages/next/src/build/webpack-config.ts#L180